src/client/camera.cpp

```diff
@@ -60,6 +60,8 @@ Camera::Camera(MapDrawControl &draw_control, Client *client):
 	m_wieldnode->setItem(ItemStack(), m_client);
 	m_wieldnode->drop(); // m_wieldmgr grabbed it
 
+	m_camera_mode_restrictions = new CameraModeRestrictions();
```

Comment:
memleak, this value is never delete. I suggest to use stack instead of pointer here, due to the usecase.

Reply:
Done what I think you suggested. :)